### PR TITLE
[meteoalerte] Adressing Issue #8543

### PR DESCRIPTION
**Pull request by clinique**
Signed-off-by: clinique <gael@lhopital.org>

### CODE REVIEW

**Comment by clinique** [comment]
Issue #8543

**Review by lolodomo — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java [hunk 1]
```diff
@@ -168,12 +165,14 @@ public void updateAlert(String channelId, AlertLevel value) {
             updateState(channelId, getAlertLevel(value));
         }
         if (isLinked(channelIcon)) {
-            String resource = getResource(String.format("picto/%s.svg", channelId));
-            if (resource != null) {
+            State result = UnDefType.UNDEF;
+            byte[] bytes = getResource(String.format("picto/%s.svg", channelId));
+            if (bytes != null) {
+                String resource = new String(bytes);
```
Comment (cpmeister) [inline bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java:171]: Please specify the charset used here

File: bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java [hunk 2]
```diff
@@ -168,12 +165,14 @@ public void updateAlert(String channelId, AlertLevel value) {
...
                 resource = resource.replaceAll(UNKNOWN_COLOR, ALERT_COLORS.getOrDefault(value, UNKNOWN_COLOR));
+                result = new RawType(resource.getBytes(), "image/svg+xml");
```
Comment (cpmeister) [inline bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java:173]: and here as well

**Review by cpmeister — CHANGES_REQUESTED** [review]

**Review by cpmeister — APPROVED** [review]

**Comment by cpmeister** [comment]
Strange, looks like the vigicrues build is failing. 

```
[ERROR] org.openhab.binding.vigicrues.internal.dto.hubeau.HubEauResponse.java:[1]
Header is missing
[ERROR] org.openhab.binding.vigicrues.internal.dto.hubeau.HubEauResponse.java:[7]
An author tag is missing
```

It isn't related to this PR but it is something that you should fix in another PR if it is still an issue.

**Pull request merged**